LCM/dsc/engine/ConsistencyInvoker/ConsistencyInvoker.c

```diff
@@ -70,15 +70,15 @@ char* getPythonProvider()
         strcat(result,buffer);
     }
 
-    // If python2 --version does not contain 'not found' return python2
-    if(strstr(result, "not found") == NULL) {
-    	return PYTHON2_COMMAND;
+    // If python3 --version exists
+    if(*result != '\0' && result[0] == '3') {
+    	return PYTHON3_COMMAND;
     }
 
     // Look for python3
```

Comment:
Please update comment to reflect the change

Reply:
this is done